Source/WebKit/WebProcess/WebPage/RemoteLayerTree/PlatformCALayerRemote.mm

```diff
@@ -1101,5 +1101,11 @@ static inline bool isEquivalentLayer(const PlatformCALayer* layer, const std::op
         return m_properties.backingStoreOrProperties.store->purgeBackBufferForTesting();
 }
 
+void PlatformCALayerRemote::markFrontBufferVolatileForTesting()
+{
+    if (m_properties.backingStoreOrProperties.store)
```

Comment:
Should put the store in a CheckedPtr local before calling the member function on it (since not ref counted).